tronpy/providers/http.py

```diff
@@ -56,6 +58,11 @@ def __init__(
             self._default_api_keys = self._api_keys.copy()
         else:
             self.use_api_key = False
+        
+        if jw_token is None or isinstance(jw_token, str):
```

Comment:
For case of `js_token` being None, it shouldn't set the attribute to `"Bearer "`, but instead not add it to the headers at all

Reply:
okay, thanks.